lib/commands/network.js

```diff
@@ -38,7 +38,8 @@ commands.setNetworkConnection = async function (type) {
   });
   if (!airplaneMode) {
     await this.wrapBootstrapDisconnect(async () => {
-      await this.adb.setWifiAndData({wifi, data});
+      await this.adb.setWifiState(wifi);
+      await this.adb.setDataState(data);
```

Comment:
@imurchie Im reverting this as these methods doesnt take objects as inputs here,https://github.com/appium/appium-adb/pull/158/files#diff-83add4eba9c23315341d94aec97ce76bR291